stdlib/universe/unique.go

```diff
@@ -156,41 +157,83 @@ func (t *uniqueTransformation) Process(id execute.DatasetID, tbl flux.Table) err
 			// Check unique
 			switch col.Type {
 			case flux.TBool:
-				v := cr.Bools(colIdx).Value(i)
-				if boolUnique[v] {
-					continue
+				if vs := cr.Bools(colIdx); vs.IsNull(i) {
+					if nullUnique {
+						continue
+					}
+					nullUnique = true
+				} else {
+					v := vs.Value(i)
+					if boolUnique[v] {
+						continue
+					}
+					boolUnique[v] = true
 				}
```

Comment:
It was not necessary to check for null for the tests to work, but I thought we shouldn't rely on default values for null elements.

Reply:
That is correct.

Reply:
That is correct.